libcrux-ml-kem/src/vector/neon/compress.rs

```diff
@@ -103,6 +103,14 @@ fn decompress_uint32x4_t<const COEFFICIENT_BITS: i32>(v: _uint32x4_t) -> _uint32
     decompressed
 }
 
+#[inline(always)]
+pub fn decompress_1(a: SIMD128Vector) -> SIMD128Vector {
+    let z = ZERO();
+    let s = super::arithmetic::sub(z, &a);
+    let res = super::arithmetic::bitwise_and_with_constant(s, 1665);
+    res
```

Comment:
If this is not required for a proof, the value should be returned directly.

```suggestion
    super::arithmetic::bitwise_and_with_constant(s, 1665)
```